config/app.cfg

```diff
@@ -22,3 +22,6 @@ webhook_key = some_string
 region = us-east-1
 subscriber = 512686554592
 name = bus-profiles
+env = dev
```

Comment:
I saw `env` in the current configuration provided by the formula, but maybe it's obsolete?

Reply:
It's used when generating the `arn`:

```
return 'arn:aws:sns:{region}:{subscriber}:{name}--{env}'.format(region=region,
                                                                subscriber=subscriber,
                                                                name=name,
                                                                env=env)
```

Reply:
That's `profiles.environment`.